tls/s2n_tls.h

```diff
@@ -44,8 +44,8 @@ extern int s2n_server_cert_verify_send(struct s2n_connection *conn);
 extern int s2n_server_cert_verify_recv(struct s2n_connection *conn);
 extern int s2n_server_key_send(struct s2n_connection *conn);
 extern int s2n_server_key_recv(struct s2n_connection *conn);
-extern int s2n_client_cert_req_recv(struct s2n_connection *conn);
-extern int s2n_client_cert_req_send(struct s2n_connection *conn);
+extern int s2n_server_cert_req_recv(struct s2n_connection *conn);
+extern int s2n_server_cert_req_send(struct s2n_connection *conn);
```

Comment:
Talked offline: "client_cert_req" and "server_cer_req" both are arguably confusing in their own ways, so we settled on just "s2n_cert_req_"